shared/menubar/chat-container.desktop.tsx

```diff
@@ -21,13 +21,15 @@ const mapDispatchToProps = dispatch => ({
 })
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => ({
-  convRows: stateProps.conversations
-    .slice(0, ownProps.convLimit ? ownProps.convLimit : stateProps.conversations.length)
-    .map(c => ({
-      conversationIDKey: c.conversationIDKey,
-      onSelectConversation: () => dispatchProps._onSelectConversation(c.conversationIDKey),
-      ...c,
-    })),
+  convRows: __STORYBOOK__
```

Comment:
i didn't fixup remote connect to participate in this so for now this makes it work